groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/lanthanides/separations/XenotimeProcessing.groovy

```diff
@@ -53,7 +53,7 @@ MACERATOR.recipeBuilder()
 ELECTROMAGNETIC_SEPARATOR.recipeBuilder()
     .inputs(ore('dustXenotime'))
     .outputs(metaitem('dustConcentrateXenotime'))
-    .chancedOutput(metaitem('dustMonazite'), 2500, 0)
+    .chancedOutput(metaitem('dustMonazite'), 1500, 0)
```

Comment:
starting to think we should remove xenotime alluvial deposits altogether and just get xenotime from monazite alluvial

Reply:
let's JEI hide it and remove its recipes from ore sorting bc if we remove it from susycore the enum order would freak out

Reply:
alright then